test/Worker.Extensions.DurableTask.Analyzers.Test/Verifiers/CSharpAnalyzerVerifier`1+Test.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.CodeAnalysis.CSharp.Testing;
```

Comment:
adding copyright notice :) 
```suggestion
﻿// Copyright (c) .NET Foundation. All rights reserved.
// Licensed under the MIT License. See LICENSE in the project root for license information.

﻿using Microsoft.CodeAnalysis.CSharp.Testing;
```